config/locales/en.models.yml

```diff
@@ -252,6 +252,7 @@ en:
         user_notes_field_mode: Users may submit a note when purchasing?
         user_notes_label: Label for Notes Field
         order_notification_recipient: Order Notification Recipient
+        cancellation_notification_contacts: Cancellation Notification Contacts
```

Comment:
I think we should call this `Recipient`, to be consistent with the other fields on this page, and we should probably change it to be plural (`Recipients`) on all fields?

Reply:
I wasn't sure which way to go. We have "Training Request Contacts" and "Order Notification Recipient". Making this change definitely highlighted the fact that those two are inconsistent functionally: training requests allows a comma separated list of emails while order notification only allows one. I felt changing that behavior was outside the scope of this.

I think I do prefer Recipients over Contacts. I'll make that change, but probably leave "Order Notification Recipient" singular.